File: testRes.js
function testResult(array) {
    let avg = (array.reduce((a,b)=> a + b, 0) / array.length).toFixed(3);
    let freqDict = {'h':0, 'a':0, 'l':0};
    for (let x of array){
      if (x > 8){
        freqDict['h']++;
      }else if (x > 6){
        freqDict['a']++;
      }else{
        freqDict['l']++;
      }
    }
   if (freqDict['h'] == array.length){
     return [Number(avg), freqDict, 'They did well'];
   }else{
     return [Number(avg), freqDict];
   }
  }

File: test_res.py
def test(r):
    avg = round((sum(r)/len(r)), 3)
    test_dict = {'h':0, 'a':0, 'l':0}
    for x in r:
        if x > 8:
            test_dict['h'] += 1
        elif x > 6:
            test_dict['a'] += 1
        else:
            test_dict['l'] += 1
    if test_dict['h'] == len(r):
        return [avg, test_dict, 'They did well']
    else:
        return [avg, test_dict]
    